HW9/task-1.py

```diff
@@ -0,0 +1,86 @@
+# Напишите следующие функции:
+# Нахождение корней квадратного уравнения
+# Генерация csv файла с тремя случайными числами в каждой строке. 100-1000 строк.
+# Декоратор, запускающий функцию нахождения корней квадратного уравнения
+# с каждой тройкой чисел из csv файла.
+# Декоратор, сохраняющий переданные параметры и результаты работы функции в json файл.
+import csv
+from random import randint as r
+from typing import Callable
+import json
+from pathlib import Path
+from functools import wraps
+
+
+def deco_read(func1: Callable) -> None:
+    @wraps(func1)
+    def wrapper_a(*args):
+        with open("number_tab.csv", 'r', newline='') as f:
+            csv_file = csv.reader(f)
+            for i, line in enumerate(csv_file):
+                if i == 0:
+                    continue
+                else:
+                    row = ''.join(line)
+                    params = row.split()
+                    params = [int(j) for j in params]
+                    result = func1(params)
```

Comment:
Зачем столько сложностей
```suggestion
                    result = func1(*map(int, line))
```